org.mwc.debrief.core/src/org/mwc/debrief/core/actions/ExportAsGeoPDFHandler.java

```diff
@@ -32,51 +48,54 @@ protected void execute() {
 			loadBackgroundLayers(theLayers, configuration);
 			configuration.setViewportArea(theChart.getProjectionArea());
 			configuration.setLandscape(theChart.getScreenSize().getWidth() > theChart.getScreenSize().getHeight());
-			
+
 			final FileDialog dialog = new FileDialog(getShell(), SWT.SAVE);
 			dialog.setFilterNames(new String[] { "PDF Files", "All Files (*.*)" });
 			dialog.setFilterExtensions(new String[] { "*.pdf", "*.*" });
-			
+
 			final String userFileName = dialog.open();
 			if (userFileName != null && !userFileName.isEmpty()) {
 				configuration.setPdfOutputPath(userFileName);
 				GeoPDF geoPdf = GeoPDFBuilder.build(theLayers, configuration);
+				Application.logError3(ToolParent.INFO,
+						"GeoPDF Compose files, background and environment are ready to be compiled.", null, false);
 				GeoPDFBuilder.generatePDF(geoPdf, configuration);
 			}
 
 		} catch (Exception e) {
 			final MultiStatus status = createMultiStatus(e.getLocalizedMessage(), e);
-			
+
 			ErrorDialog.openError(getShell(), "Error", e.getMessage(), status);
 		}
 
 	}
-	
+
 	private static MultiStatus createMultiStatus(String msg, Throwable t) {
 
-        final List<Status> childStatuses = new ArrayList<>();
-        final StackTraceElement[] stackTraces = t.getStackTrace();
+		final List<Status> childStatuses = new ArrayList<>();
+		final StackTraceElement[] stackTraces = t.getStackTrace();
 
-        for (StackTraceElement stackTrace: stackTraces) {
-            Status status = new Status(IStatus.ERROR,
-                    "Export GeoPDF", stackTrace.toString());
-            childStatuses.add(status);
-        }
+		for (StackTraceElement stackTrace : stackTraces) {
+			Status status = new Status(IStatus.ERROR, "Export GeoPDF", stackTrace.toString());
+			childStatuses.add(status);
+		}
 
-        final MultiStatus ms = new MultiStatus("org.mwc.debrief.core",
-                IStatus.ERROR, childStatuses.toArray(new Status[] {}),
-                t.toString(), t);
-        return ms;
-    }
+		final MultiStatus ms = new MultiStatus("org.mwc.debrief.core", IStatus.ERROR,
+				childStatuses.toArray(new Status[] {}), t.toString(), t);
+		return ms;
+	}
 
 	public void loadBackgroundLayers(final Layers theLayers, final GeoPDFConfiguration configuration) {
 		final Enumeration<Editable> enume = theLayers.elements();
-		while(enume.hasMoreElements()) {
+		while (enume.hasMoreElements()) {
 			final Editable currentEditable = enume.nextElement();
 			if (currentEditable instanceof WorldImageLayer) {
 				final WorldImageLayer tif = (WorldImageLayer) currentEditable;
 				if ("GeoTiff".equals(tif.getDataType()) && new File(tif.getFilename()).exists()) {
```

Comment:
"GeoTiff" is a `magic string`.

Could you use the public const?  
![image](https://user-images.githubusercontent.com/1108513/93475810-f9805c80-f8f0-11ea-959b-aa12f3f08bdb.png)